docs/instrumentation_example.md

```diff
@@ -1,4 +1,4 @@
-# Probes and Meters: Examples for Adding to Code
+# Armory Instrumentation Examples: Capture Artifacts from Existing Code
```

Comment:
I might change `Capture Artifacts from Existing Code` to `Measuring Experiment Artifacts Using Probes and Meters`. And `instrumentation_example.md` --> `instrumentation_examples.md`. But otherwise this commit looks good

Reply:
resolved with ee52a9e7